airqo-platform/AirQo-frontend
# boiler plate code and structuring files using the ducks pattern

- What does this PR do?
This PR has boiler plate code with structuring of files to enforce the ducks pattern
- What JIRA task is related to this PR?
There is not JIRA card for this.
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm start` to test out the development server
- Which end points should are ready for testing?
There are no end points for this PR
